hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMock.java

```diff
@@ -0,0 +1,93 @@
+package pl.allegro.tech.hermes.mock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+public class HermesMock {
```

Comment:
Every field here should be final. It helps to read code, because you instantly know that this cannot change. Check also another classes like HermesMockDefine etc.